### PR TITLE
core: bind services to localhost

### PR DESCRIPTION
**Pull request by InTheCloudDan**

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -15,17 +15,13 @@ done
...
 # Log into Production Server, Pull and Restart Docker
 ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose pull'
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose build'
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose down --remove-orphans'
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose up -d'
-
-# Restart Nginx 
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'sudo nginx -s reload'
+#ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose build'
+#ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose down --remove-orphans'
```
Comment (levlaz) [inline scripts/deploy.sh]: We will eventually run out of disk space if we don't do this step. Any reason why you removed it?

Comment (levlaz) [reply scripts/deploy.sh]: @InTheCloudDan did you see this comment?

Comment (InTheCloudDan) [reply scripts/deploy.sh]: @levlaz this is outdated now. I think we should add `-a` to the docker system prune command and we'd be all set.

File: scripts/deploy.sh [hunk 2]
```diff
@@ -15,17 +15,13 @@ done
...
 
 # Log into Production Server, Pull and Restart Docker
 ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose pull'
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose build'
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose down --remove-orphans'
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose up -d'
-
-# Restart Nginx 
-ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'sudo nginx -s reload'
+#ssh -o StrictHostKeyChecking=no $PRODUCTION_SERVER 'cd /var/www/app && docker-compose build'
```
Comment (levlaz) [inline scripts/deploy.sh]: @InTheCloudDan can we get rid of these comments before we merge?

**Pull request merged**